spring-cloud-gcp-samples/spring-cloud-gcp-pubsub-stream-dead-letter-sample/pom.xml

```diff
@@ -63,6 +63,10 @@
             <version>2.11.0</version>
             <scope>test</scope>
         </dependency>
+        <dependency>
+            <groupId>org.springframework</groupId>
+            <artifactId>spring-messaging</artifactId>
+        </dependency>
```

Comment:
This dependency is already brought in transitively -- Spring Cloud Stream is built on Spring Integration.

You can verify this by running the dependency tree in the stream binder module or the sample app:
```
mvn dependency:tree -Dincludes=org.springframework:spring-messaging
```